src/Pages/CartList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { useStore } from '../Store/context'
+import CartItem from '../Cart/CartItem';
+import "./CartList.css"
+import CartTotal from '../Cart/CartTotal';
+import Toast from '../Components/Toast';
+
+const CartList = () => {
+    const {state} = useStore()
+
+    return (
+        <div className="cartContainer">
+        {state.isLoading === "removing" ? <Toast mesg="Removing from Cart"/> : null}
+            <h1>Cart</h1>
+                {state.cart.length === 0 ? <p>Your Shopping bag is empty!</p> : (
```

Comment:
1 line ternary operators are better. This can be moved to if else since returning components